internal/rootcoord/garbage_collector.go

```diff
@@ -2,25 +2,18 @@ package rootcoord
 
 import (
 	"context"
-	"time"
-
-	"github.com/milvus-io/milvus/api/commonpb"
-	ms "github.com/milvus-io/milvus/internal/mq/msgstream"
-	"github.com/milvus-io/milvus/internal/proto/internalpb"
 
 	"github.com/milvus-io/milvus/internal/util/typeutil"
 
-	"github.com/milvus-io/milvus/internal/log"
 	"github.com/milvus-io/milvus/internal/metastore/model"
-	"go.uber.org/zap"
 )
 
 type GarbageCollector interface {
 	ReDropCollection(collMeta *model.Collection, ts Timestamp)
 	RemoveCreatingCollection(collMeta *model.Collection)
 	ReDropPartition(pChannels []string, partition *model.Partition, ts Timestamp)
-	GcCollectionData(ctx context.Context, coll *model.Collection, ts typeutil.Timestamp) error
-	GcPartitionData(ctx context.Context, pChannels []string, partition *model.Partition, ts typeutil.Timestamp) error
+	GcCollectionData(ctx context.Context, coll *model.Collection, ts typeutil.Timestamp) (ddlTs Timestamp, err error)
+	GcPartitionData(ctx context.Context, pChannels []string, partition *model.Partition, ts typeutil.Timestamp) (ddlTs Timestamp, err error)
 }
 
 type GarbageCollectorCtx struct {
```

Comment:
Ctx? a bit of confusing and misunderstanding

Reply:
How about `bgGarbageCollector`?